microservices-orders/services-start.sh

```diff
@@ -1,7 +1,10 @@
 #!/bin/bash
 
+# Source Confluent Platform versions
+. ../utils/config.sh
```

Comment:
My original thought when I suggested we have helper.sh source in config.sh, we just have all our scripts that users would run, just source in helper.sh.  That way it's just standard and the same across all scripts.  WDYT?

Reply:
Given the move to config.env, I think current PR works.  Is that right or did I misunderstand your comment?

Reply:
i'm thinking this should be the following, just to standardize

```suggestion
. ../utils/helper.sh
```

Reply:
config.sh isn't a file anymore after the change to config.env

Reply:
Great catch!